edi_oca/static/description/index.html

```diff
@@ -8,11 +8,10 @@
 
```

Comment:
TIP for the future: you can use `export SKIP=oca-gen-addon-readme` in your shell to avoid this noise  and possible conflicts :wink: 
See https://github.com/OCA/edi-framework/blob/16.0/.github/workflows/pre-commit.yml#L38

Reply:
I cleaned up the commits. Will backport  again to https://github.com/OCA/edi/pull/1029 once merged